example/operator/20-garden.yaml

```diff
@@ -77,18 +77,26 @@ spec:
       main:
         backup:
           provider: local
-        # providerConfig:
-        #   <some-provider-specific-config-for-the-backup-buckets>
+    #     bucketName: gardener-operator # if not provided, gardener-operator generates a name
+    #     providerConfig:
+    #       <some-provider-specific-config-for-the-backup-buckets>
           secretRef:
             name: virtual-garden-etcd-main-backup-local
-        # bucketName: gardener-operator # if not provided, gardener-operator generates a name
```

Comment:
Nit: Why did you change this?

Reply:
Probably an rebase issue, thanks for noticing.

Reply:
It was actually on purpose to move `bucketName` and `providerConfig` (both commented) together which is visually nicer. Still an issue? 😃